spec/lib/mvi/service_spec.rb

```diff
@@ -281,9 +281,6 @@
       let(:base_path) { MVI::Configuration.instance.base_path }
       it 'should raise a service error', :aggregate_failures do
         allow_any_instance_of(Faraday::Connection).to receive(:post).and_raise(Faraday::TimeoutError)
-        expect(Rails.logger).to receive(:warn).with(
```

Comment:
Is this mostly checking if the certificate was checked?

I've got very little context too, but this is expecting to fail to load a certificate after a certificate AND to do a gateway timeout? Seems like two pretty different things. 

If the test on the certificate check was necessary, i certainly wouldn't rely on the `Rails.logger` to log a message -- I'd expect to get an exception from whatever is doing the checking.

Reply:
> Is this mostly checking if the certificate was checked?

No. It was a hack just to get a spec to pass, after we changed a Rails.logger level from `:error` to `:warn' - context: #3247. I'm not convinced that it ever caused the spec to pass in the first place but I still can't explain why...